src/lib/QtWidgets/QTreeWidgetItem.ts

```diff
@@ -0,0 +1,38 @@
+import addon from '../utils/addon';
+import { Component, NativeElement } from '../core/Component';
+import { QTreeWidget } from '../..';
+
+export class QTreeWidgetItem extends Component {
+    native: NativeElement;
+
+    constructor();
+    constructor(strings?: string[]); // 1
+    constructor(parent?: QTreeWidget); // 2
+    constructor(parent?: QTreeWidget, strings?: string[]); // 3
+    constructor(parent?: QTreeWidget, preceding?: QTreeWidgetItem); // 3
+    constructor(parent?: QTreeWidgetItem); // 4
+    constructor(parent?: QTreeWidgetItem, strings?: string[]); // 5
+    constructor(parent?: QTreeWidgetItem, preceding?: QTreeWidgetItem); // 6
+    constructor(other?: QTreeWidgetItem); // 9
+    // The last parameter below corresponds to the constructor above.
+    constructor(arg: any = null) {
```

Comment:
Also, a constructor of any type would actually defeat the purpose of having typescript typecheck the constructor since it would pass all cases.

Reply:
Thanks for the tips, I am still learning haha.